.github/workflows/docker/Dockerfile-compilation-testing-anolis8.4

```diff
@@ -0,0 +1,32 @@
+FROM openanolis/anolisos:8.4-x86_64
+
+LABEL maintainer="Shirong Hao <shirong@linux.alibaba.com>"
+
+RUN dnf clean all && rm -r /var/cache/dnf && \
+    dnf --enablerepo=PowerTools install -y git wget \
+    make cmake autoconf libtool  gcc gcc-c++ \
+    openssl-devel libcurl-devel yum-utils \
+    python2 bzip2 perl which && \
+    alternatives --set python /usr/bin/python2
+
+WORKDIR /root
+
+# install rust
+RUN curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y --no-modify-path
+ENV PATH         /root/.cargo/bin:$PATH
+
+
+# install SGX
```

Comment:
please use the env to creating a global const for sgx SDK version. 

ENV SGX_SDK_VERSION         2.15.1
ENV SGX_SDK_RELEASE_NUMBER  2.15.101.1